Mathlib/NumberTheory/Cyclotomic/Three.lean

```diff
@@ -82,7 +82,7 @@ private lemma lambda_sq : λ ^ 2 = -3 * η := by
   _ = -3 * η := by ring
 
 /-- We have that `η ^ 2 = -η - 1`. -/
-private lemma eta_sq : (η ^ 2 : 𝓞 K) = - η - 1 := by
+lemma eta_sq : (η ^ 2 : 𝓞 K) = - η - 1 := by
```

Comment:
Why are you finaling removing the private attribute?

Reply:
Because I need it in the other file.